nornir/plugins/inventory/netbox.py

```diff
@@ -36,17 +36,41 @@ def __init__(
         )
         headers = {"Authorization": "Token {}".format(nb_token)}
 
+        def make_request(url: str) -> Any:
+            req = requests.get(url=url, headers=headers, params=filter_parameters)
+            req.raise_for_status()
+
+            if req.ok:
```

Comment:
Do you need this given you did raise_for_status?

Reply:
Pretty sure `raise_for_status()` doesn't catch everything.

Reply:
Ok, in that case I think you should raise an exception if `req.ok` returns `False`.